content/en/network_monitoring/performance/network_analytics.md

```diff
@@ -21,7 +21,11 @@ further_reading:
       text: 'Collect your Network Data with the Datadog Agent.'
 ---
 
-{{< img src="network_performance_monitoring/network_analytics/main_page_npm2.png" alt="Main page" >}}
+## Overview
+
+The Network Analytics page provides insights into your overall network health and shows [recommended queries](#recommended-queries) at the top of the page. These recommended queries provide simplified access to run common queries, and provide a snapshot of related metrics to the query so that you can see relevant changes in throughput, latency, DNS errors, and more. Clicking on a recommended query automatically populates the search bar, group bys, and summary graphs into a unique combination to provide you with the most relevant insights into your network.
+
+{{< img src="network_performance_monitoring/network_analytics/main_page_npm3.png" alt="Main page" >}}
```

Comment:
I realize this line is just moved, not added, but I think it's a good opportunity to add more descriptive alt text.

Reply:
great catch lol.. Updated that :)